explain ABI in rust blogpsot (#25)
metalbear-co/metalbear.co: content/en/blog/hooking-go-from-rust/index.md

Feel free to checkout [mirrord](https://github.com/metalbear-co/mirrord), send corrections/issues with the blog post on our [website’s repository](https://github.com/metalbear-co/metalbear.co) or just reach us at hi@metalbear.co.

> [^1]: Application binary interface - how binary programs interact, how do they use the stack, registers, etc.

Reviewer: ```suggestion
[^1]: Application Binary Interface - an application's interface at the binary level: how it uses the stack, registers, etc.
```

[^2]: The cool Gopher was made using https://gopherize.me/.

> [^3]: Foreign Function Interface - How to interact with external functions provided by other frameworks/dependencies.

Reviewer: ```suggestion
[^3]: Foreign Function Interface - how a language/framework interacts with external functions provided by other languages/frameworks.
```
